src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -745,10 +744,10 @@ void M()
 
             SwitchStatementSyntax switchStmt = methodDecl.Body.Statements[1];
             SwitchLabelSyntax switchLabel = switchStmt.Sections[0].Labels[0];
-            var symbol1 = model.GetDeclaredSymbol(switchLabel);
+            var symbol1 = (ISymbol)model.GetDeclaredSymbol(switchLabel);
             Assert.NotNull(symbol1);
-            Assert.IsAssignableFrom<SourceLabelSymbol>(symbol1);
```

Comment:
Why was this assert removed?

Reply:
> Why was this assert removed?

It doesn't add any value, the cast below is going to fail as well. 

---
In reply to: [339251845](https://github.com/dotnet/roslyn/pull/39498#discussion_r339251845) [](ancestors = 339251845)